src/cc65/declare.c

```diff
@@ -2089,14 +2089,15 @@ static unsigned ParseStructInit (Type* T, int AllowFlexibleMembers)
 
             /* Account for the data and output it if we have a full word */
```

Comment:
The comment should say, "`have full bytes.`".

Reply:
Done.